grammars/html.cson

```diff
@@ -447,7 +447,7 @@
       }
     ]
   'tag-generic-attribute':
-    'match': '(?<=[^=])\\b([a-zA-Z0-9:-]+)'
+    'match': '(?<=[^=]|^)\\b([a-zA-Z0-9:-]+)'
```

Comment:
Can you try changing this to `(?<!=)` instead?  Because right now it's doing a positive lookbehind for anything that _doesn't_ match `=`...so a negative lookbehind for `=` should do the same thing and still fix #157.